docs/DeveloperGuide.adoc

```diff
@@ -64,7 +80,9 @@ image::LogicClassDiagram.png[]
 [discrete]
 ==== How the architecture components interact with each other
 
-The _Sequence Diagram_ below shows how the components interact with each other for the scenario where the user issues the command `delete 1`.
+This section explains the interactions between the different architecture components through an example.
```

Comment:
"the interactions **amongst** the different architecture components **using** an example."

Reply:
Changed